test/tests/config/ci.py

```diff
@@ -1,3 +1,7 @@
 from .development import *  # noqa
```

Comment:
Could be worth checking if @Rondineli needs to re-run his Chef config scripts for these config files